packages/web-react/src/components/FieldGroup/__tests__/FieldGroup.test.tsx

```diff
@@ -0,0 +1,105 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+import React from 'react';
+import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+import { validationTextPropsTest } from '../../../../tests/providerTests/validationTextPropsTest';
+import { validationStatePropsTest } from '../../../../tests/providerTests/dictionaryPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import FieldGroup from '../FieldGroup';
+
+describe('FieldGroup', () => {
+  const itemList = (
+    <>
+      <div>Item</div>
+      <div>Item</div>
+      <div>Item</div>
+    </>
+  );
+
+  classNamePrefixProviderTest(FieldGroup, 'FieldGroup');
+
+  validationStatePropsTest(FieldGroup, 'FieldGroup--');
+
+  validationTextPropsTest(FieldGroup, '.FieldGroup__validationText');
+
+  stylePropsTest(
+    (props) => <FieldGroup {...props} label="Label" id="FieldGroupExample" data-testid="test-FieldGroup" />,
+    'test-FieldGroup',
+  );
+
+  restPropsTest((props) => <FieldGroup {...props} label="Label" />, 'fieldset');
+
+  it('should render items as children', () => {
+    const dom = render(<FieldGroup label="Label">{itemList}</FieldGroup>);
+
+    const fieldGroup = dom.container.querySelector('fieldset') as HTMLElement;
+    expect(fieldGroup).toHaveClass('FieldGroup');
```

Comment:
I would prefer writing an empty line before `expect` as we do it with the `return` statement. I am suggesting this for consistency with other tests and for better readability.

Reply:
added empty line before, but we still have inconsistency in other tests, somewhere are empty lines, somewhere not.

Reply:
I am ware of that and trying to refactor it.